src/pages/index.vue

```diff
@@ -57,28 +58,22 @@
       <!-- Width is 57.143vh i.e. half of height (because grid dimensions are 4 ⨯ 2) -->
       <div
         class="homepage-images flex flex-row gap-4 lg:gap-0 items-center lg:grid lg:grid-cols-2 lg:grid-rows-4 lg:w-[57.143vh] lg:h-[114.286vh]"
+        aria-hidden
       >
-        <Transition
-          v-for="(image, index) in featuredSearchImages"
+        <NuxtLink
+          v-for="(image, index) in featuredSearch.images"
           :key="image.identifier"
-          name="fade"
-          mode="out-in"
-          appear
+          :to="image.url"
+          class="homepage-image block aspect-square h-30 w-30 lg:h-auto lg:w-auto lg:m-[2vh] rounded-full"
+          :style="{ '--transition-index': `${index * 0.05}s` }"
         >
```

Comment:
Should we preserve the one-by-one fade-in animation? That might still look quite nice on the static page?

Reply:
Sure thing.